### PR TITLE
fix: Broken Docker Builds

### PR DESCRIPTION
**Pull request by sojan-official**
fix the broken docker builds due to #2737

### CODE REVIEW

**Review by vishnu-narayanan — DISMISSED** [review]

File: docker/Dockerfile [hunk 1]
```diff
@@ -83,9 +83,9 @@ COPY --from=pre-builder /gems/ /gems/
 COPY --from=pre-builder /app /app
 
 # Remove unnecessary files
-RUN rm -rf /gems/ruby/3.0.2/cache/*.gem \
-  && find /gems/ruby/3.0.2/gems/ -name "*.c" -delete \
-  && find /gems/ruby/3.0.2/gems/ -name "*.o" -delete
+RUN rm -rf /gems/ruby/3.0.0/cache/*.gem \
+  && find /gems/ruby/3.0.0/gems/ -name "*.c" -delete \
+  && find /gems/ruby/3.0.0/gems/ -name "*.o" -delete
```
Comment (paescuj) [inline docker/Dockerfile:88]:
```suggestion
  && find /gems/ruby/3.0.0/gems/ \( -name "*.c" -o -name "*.o" \) -delete
```

Comment (paescuj) [reply docker/Dockerfile:88]: Thanks! Was about to submit the same fix 👍 Just suggesting a small change to combine the two `find` commands into one.

Comment (sojan-official) [reply docker/Dockerfile:88]: Nice catch 😃

**Comment by sojan-official** [comment]
verifying https://github.com/chatwoot/chatwoot/issues/2609 before merging

**Pull request merged**